### PR TITLE
Line-ending test

### PR DESCRIPTION
**Pull request by ad-daniel**
**Description**
Added test that ensures line-ending conformity for the included assets.

### CODE REVIEW

**Comment by ad-daniel** [comment]
The reason it was failing on Windows it that it seems git automatically converts to CRLF when cloning, which makes it fail the test. So for windows we need to specifically change git's config before the checkout action occurs.

File: tests/sources/test_line_ending.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+class TestLineEnding(unittest.TestCase):
+    """Unit test of the line endings"""
+
+    def setUp(self):
+        """Get all files to be tested"""
+
+        self.files = []
+        for root_path, _, file_names in os.walk(os.environ['WEBOTS_HOME']):
+            for extension in ('*.proto', '*.wbt', '*.wbo', '*.forest'):
```
Comment (omichel) [inline tests/sources/test_line_ending.py:31]: Why not adding more text files here? Like the *.c, *.h, *.cpp, *.hpp, *.py, *.java, *.m, *.md, etc.

**Review by omichel — APPROVED** [review]
Thank you.

**Review by omichel — APPROVED** [review]
Thank you.

**Pull request merged**